README.md

```diff
@@ -144,29 +144,12 @@ First let's identify your Azure subscription and save it for use later on in the
    az provider register -n Microsoft.ContainerInstance
    ```
 
-## Quick set up with AKS
+## Quick set up with AKS (Linux only)
+Azure Kubernetes Service has an efficient way of setting up virtual kubelet with the ACI provider with a feature called virtual node. You can easily install a virtual node that will deploy Linux workloads to ACI. The pods that spin out will automatically get private IPs and will be within a subnet that is within the AKS cluster's Virtual Network. **Virtual Nodes is the recommended path for using the ACI provider on Linux AKS clusters.** Virtual node does not support Windows workloads at this time. 
 
-### Linux containers with Virtual Nodes
-
-Azure Kubernetes Service has an efficient way of setting up virtual kubelet with the ACI provider with a feature called virtual node. You can easily install a virtual node that will deploy Linux workloads to ACI. The pods that spin out will automatically get private IPs and will be within a subnet that is within the AKS cluster's Virtual Network. **Virtual Nodes is the recommended path for using the ACI provider on Linux AKS clusters.** 
-
-To install virtual node in the Azure portal go [here](https://docs.microsoft.com/azure/aks/virtual-nodes-portal). To install virtual node in the Azure CLI go [here](https://docs.microsoft.com/azure/aks/virtual-nodes-cli). 
-
-### Windows containers
-
-The virtual nodes experience does not exist for Windows containers yet including no virtual networking support. You can use the following command to install the ACI provider for Windows. 
-
-**Bash**
-```cli 
-az aks install-connector --resource-group <aks cluster rg> --name <aks cluster name> --os-type windows 
-```
-
-Once created, [verify the virtual node has been registed](#Validate-the-Virtual-Kubelet-ACI-provider) and you can now [schedule pods in ACI](#Schedule-a-pod-in-ACI). 
-
-:bangbang: The _az aks install-connector_ command will be deprecated shortly. Please follow the **Manual set-up** section below instead. :bangbang:
-
-## Manual set-up
+To install virtual node in the Azure portal go [here](https://docs.microsoft.com/azure/aks/virtual-nodes-portal). To install virtual node in the Azure CLI go [here](https://docs.microsoft.com/azure/aks/virtual-nodes-cli).
```

Comment:
Make sure to use **Virtual Node** this line and a few lines up x 3 :-) Caps and plural/singular

Reply:
Updated so all references are **Virtual Node** except for URL links which for some reason have plural **virtual-nodes**